client/webserver/site/src/html/forms.tmpl

```diff
@@ -544,3 +544,18 @@
   </div>
 </div>
 {{end}}
+
+{{define "toggleWalletStatusConfirm"}}
+<div class="form-closer hoverbg"><span class="ico-cross"></span></div>
+<div class="px-2 py-1 text-center position-relative fs22 sans-light">
+   <span id="toggleWalletStatus" class="d-hide"></span>
```

Comment:
About this last change, I'm confused why you chose to do this with a javascript locale const but three lines down you did the spans with the html template localizations and show/hide the intended span. 
Is there a reason? Because the inconsistency stands out.

Reply:
> Also we already have separate translations for in the html template strings vs. js locale consts.

I'm now utilizing the translations in `locale.go`.